rancher/charts
# WIP: added timestamp to rke2 control plane logs

https://github.com/rancher/rancher/issues/32917

# Notes
While researching this I discovered a few additional issue and fixed them:
- we were not recording rke2-agent logs
- we were recording kube-system pod logs twice
- our fluentbit service configuration did not match the other instances from banzai cloud

# Problem
The rke2 log aggregator does not include the time the log was recorded.  This makes sorting and determining when events happened impossible.

# Solution
We inject a timestamp with a lua script.  This will record when the messaged was ingested by fluentbit, giving us a timeline for when messages were logged.  

# Testing
1. install rancher logging on rke2
2. configure a cluster flow and output
3. observe the journald messages with the rke2-server.service or rke2-agent.service now have a timestamp field
